tests/integration/test_cube.py

```diff
@@ -14,14 +14,12 @@
     _UnweightedCountMeasure,
     _WeightedCountMeasure,
 )
-from cr.cube.dimension import _ApparentDimensions, AllDimensions
+from cr.cube.dimension import AllDimensions, _ApparentDimensions
 from cr.cube.enums import DIMENSION_TYPE as DT
 
-from ..fixtures import (
-    CR,  # cube-response
-    OL,  # overlaps
-    NA,  # numeric-array
-)
+from ..fixtures import CR  # cube-response
+from ..fixtures import NA  # numeric-array
+from ..fixtures import OL  # overlaps
```

Comment:
These comments should go above so these don't break onto separate redundant lines. Or maybe we just get rid of the comments altogether.